iceoryx_hoofs/test/moduletests/test_unit_duration.cpp

```diff
@@ -171,7 +179,9 @@ TEST(Duration_test, ConstructFromTimespecWithZeroValue)
     constexpr uint64_t NANOSECONDS{0U};
     constexpr Duration EXPECTED_DURATION = createDuration(SECONDS, NANOSECONDS);
 
-    struct timespec ts;
+    struct timespec ts
+    {
```

Comment:
Could we write here `struct timespec ts = {};`. This here is just confusing. I was wondering why you here redefine a system struct until I realized, no you initializing it here.

Or we have to adjust `clang-format` here, but this looks wrong.

Applies also to the lines below.

Reply:
Yes, that's how I wrote it but clang-format reformatted it this way. I'll check if I can change clang-format without reformatting the whole code. If this is too intrusive, are you okay to add this to the TODO section of the issue and adjust clang-format once we are done with clang-tidy?